### PR TITLE
Fix #2871 - remove existing pot files (for Windows)

### LINKED ISSUE #2871: Add a mecanism to remove a temp file
Would it be possible to add something to delete the {package}-client.pot in the `local` of the package (if this file already exists), before creating it.

I quite often get the error here under and manually deleting the file resolves the problem, but I would prefer if the build system would take care of that....

```
WindowsError: [Error 183] Cannot create a file when that file already exists
CONST_Makefile:708: recipe for target 'sitn/locale/sitn-client.pot' failed
make: *** [sitn/locale/sitn-client.pot] Error 1
```

### PR DESCRIPTION
**Pull request by kalbermattenm**
Please review

### CODE REVIEW

**Comment by kalbermattenm** [comment]
https://travis-ci.org/camptocamp/c2cgeoportal/builds/218002124#L7367-L7368

I do not get what this has to do with this PR... ?

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 1]
```diff
@@ -695,6 +695,7 @@ $(PACKAGE)/locale/$(PACKAGE)-$(L10N_SERVER_POSTFIX).pot: \
 		lingua-server.cfg \
 		$(VENV_BIN)/pot-create$(PYTHON_BIN_POSTFIX) \
 		$(SERVER_LOCALISATION_SOURCES_FILES)
+	rm -f $@ # Because of WindowsError, thus only problematic on Windows
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Should be after the `$(PRERULE_CMD)` command :-)

**Comment by kalbermattenm** [comment]
I applied the modifications you asked for.

By the way, thanks for the review @sbrunner

**Comment by kalbermattenm** [comment]
Travis...?

**Comment by kalbermattenm** [comment]
Thanks !

**Pull request merged**